app/src/main/java/co/nimblehq/compose/crypto/ui/theme/Color.kt

```diff
@@ -2,22 +2,20 @@ package co.nimblehq.compose.crypto.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
-object Color {
-    val White = Color(0xFFFFFFFF)
-    val Transparent = Color(0x00000000)
-    val Guyabano = Color(0xFFF8F8F8)
-    val MetallicSeaweed = Color(0xFF028090)
-    val TiffanyBlue = Color(0xFF00BFB2)
-    val Water = Color(0xFFD6F5F3)
-    val WhiteIce = Color(0xFFDBFBEC)
-    val GuppieGreen = Color(0xFF10DC78)
-    val DarkJungleGreen = Color(0xFF141B29)
-    val LightSilver = Color(0xFFD6D7D8)
-    val Quartz = Color(0xFF484D58)
-    val QuartzAlpha20 = Quartz.copy(alpha = 0.2f)
-    val FireOpal = Color(0xFFF15950)
-    val SonicSilver = Color(0xFF70747C)
-    val CaribbeanGreen = Color(0xFF00BDB0)
-    val CaribbeanGreenAlpha30 = CaribbeanGreen.copy(alpha = 0.3f)
-    val OsloGray = Color(0xFF8D9096)
-}
+val White = Color(0xFFFFFFFF)
+val Transparent = Color(0x00000000)
+val Guyabano = Color(0xFFF8F8F8)
+val MetallicSeaweed = Color(0xFF028090)
+val TiffanyBlue = Color(0xFF00BFB2)
+val Water = Color(0xFFD6F5F3)
+val WhiteIce = Color(0xFFDBFBEC)
+val GuppieGreen = Color(0xFF10DC78)
+val DarkJungleGreen = Color(0xFF141B29)
+val LightSilver = Color(0xFFD6D7D8)
+val Quartz = Color(0xFF484D58)
+val QuartzAlpha20 = Quartz.copy(alpha = 0.2f)
+val FireOpal = Color(0xFFF15950)
+val SonicSilver = Color(0xFF70747C)
+val CaribbeanGreen = Color(0xFF00BDB0)
+val CaribbeanGreenAlpha30 = CaribbeanGreen.copy(alpha = 0.3f)
+val OsloGray = Color(0xFF8D9096)
```

Comment:
Shall we rearrange it alphabetically to make it neater? 🙏🏻

Reply:
@lydiasama yes, possible 👍

Reply:
@lydiasama Resolved by https://github.com/nimblehq/jetpack-compose-crypto/pull/66/commits/100ce299291dc043ef7c811e5b12401f0cf246b1